NewEvaluation.html

```diff
@@ -654,20 +643,18 @@
 			</fieldset>
 		</form>
 
-	<div class="littleSpacing"></div>
-
-	<form  action="Evaluation.html">
-		<input  type="submit" name="button" value="Submit">
-	</form>
+		<div class= "submit-button">
+			<form action="Evaluation.html">
+				<input  type="submit" name="button" value="Submit">
```

Comment:
All the data that has been entered in the forms to be sent to the server ? and then to be sent to the Evaluation page where u can see the entred data